src/attrib.d

```diff
@@ -642,7 +723,7 @@ extern (C++) final class AnonDeclaration : AttribDeclaration
 {
 public:
     bool isunion;
-    structalign_t alignment;
+    AlignDeclaration aligndecl;
```

Comment:
This creates excessive padding in the field alignment. Swap its position with `isunion`